accounts/interface.go

```diff
@@ -0,0 +1,204 @@
+package accounts
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+const (
+	// AccountIDLen is the length of the ID that is generated as a unique
+	// identifier of an account. It is 8 bytes long so guessing is
+	// improbable, but it's still not mistaken for a SHA256 hash.
+	AccountIDLen = 8
```

Comment:
No, it's randomly assigned to use the full 8 bytes instead of an incrementing number (so it doesn't give any information about the number of accounts).